libvast/src/system/configuration.cpp

```diff
@@ -292,13 +341,6 @@ caf::error configuration::parse(int argc, char** argv) {
       put(content, key, value);
     }
   }
-  // If the user specifies a VAST_ENDPOINT, potentially use it. Precedence:
-  // 1. CLI argument
-  // 2. Environment variables
-  // 3. Config files
-  // 4. Defaults
-  if (auto vast_endpoint_env = detail::locked_getenv("VAST_ENDPOINT"))
-    caf::put(content, "vast.endpoint", *vast_endpoint_env);
```

Comment:
Unrelated to this line, but I wanted to have a resolvable comment:

Can you double-check whether we still need to have the remaining `locked_getenv` calls in the codebase? Can we maybe remove that function altogether?